codegen/smithy-aws-swift-codegen/src/main/kotlin/software/amazon/smithy/aws/swift/codegen/swiftintegrations/PackageVersionIntegration.kt

```diff
@@ -0,0 +1,24 @@
+package software.amazon.smithy.aws.swift.codegen.swiftintegrations
+
+import software.amazon.smithy.model.Model
+import software.amazon.smithy.swift.codegen.SwiftDelegator
+import software.amazon.smithy.swift.codegen.SwiftSettings
+import software.amazon.smithy.swift.codegen.core.SwiftCodegenContext
+import software.amazon.smithy.swift.codegen.integration.ProtocolGenerator
+import software.amazon.smithy.swift.codegen.integration.SwiftIntegration
+
+class PackageVersionIntegration : SwiftIntegration {
```

Comment:
Writes the package version into the service client package's Resources.  This can be used later for data collection purposes and for reference during publication.